docs/usage.md

```diff
@@ -34,9 +34,12 @@ spec:
   project: default
 ```
 
-3. Or you can pass the config-management-plugin flag to the Argo CD CLI app create command:  
+3. Or you can pass the config-management-plugin flag to the Argo CD CLI app create command:
 `argocd app create you-app-name --config-management-plugin argocd-vault-plugin`
 
+!!! warning
```

Comment:
Let’s use the bold formatting here for “warning” like we did with “important” below

Reply:
Sure thing, done.